app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -39,15 +32,43 @@ export default function NoteForm({ onClose, refetch, note, isEdit }) {
             <Input
               label="Title"
               name="title"
+              placeholder="Enter note title"
               className="w-full flex-grow-0"
               required
             />
             <Textarea
               label="Description"
               name="description"
+              placeholder="Enter note description"
+              className="w-full flex-grow-0"
+              rows={1}
+              required
```

Comment:
Implicitly `true` attributes should be kept at the top of the component's prop list.

Refer: https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=Also%20note%20that,Next